docs/index.rst

```diff
@@ -40,6 +40,19 @@ encoding
   Encoding to be used to read an OpenAPI spec. If not passed, Sphinx's
   source encoding will be used.
 
+paths
+  A comma separated list of paths to filter the included openapi spec by. For example:
```

Comment:
As I already said, I'm a bit perfectionist and may be a pain in the ass, so sorry about this. But could you please ensure that every new lines you've added no more than 80 characters in length? I believe this one and the one on line 54 are not. :)